tests/bench_sizeof_coretypes/Makefile

```diff
@@ -0,0 +1,20 @@
+include ../Makefile.tests_common
+
+# Modules that will have an impact on the size of the TCB (thread_t):
+#
+# disabled by default, enable on demand:
+ifneq (,$(USE_FLAGS))
+  USEMODULE += core_thread_flags
+endif
+#
+# enabled by default, disable on demand:
+ifneq (,$(NO_MSG))
+  DISABLE_MODULE += core_msg
+endif
```

Comment:
Was it done like this to aim to have a representative/typical size of the tcb being printed out? it would make sense to me to just have these both enabled and remove the `#ifdef`s in the code. Reasoning being that the test would have as much coverage as possible by default, as it seems to me that the main purpose of the test is to catch **any** memory usage increases. What do you think?

Reply:
Disagree: this test should allow for tracking different aspects of the core memory usage, so making the test configurable is IMHO mandatory. One can of course argue about the default configuration, but I think enabling everything that is enabled in the kernel per default (i.e. msg), and making optional kernel modules optional also in this test (i.e. thread_flags) is the way to go. Further: at the moment this test has exact the same behavior as the old `tests/sizeof_tcb` application which it supersedes - so any semantic changes should be done better in a separate PR anyway...

Reply:
ok, sound reasoning